### PR TITLE
feat: add ApisixPluginConfigs crd to v2beta3 (#746) (#638) (#791)

### PR DESCRIPTION
**Pull request by neverCase**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests
#746
#638
#791
- Target branch

### CODE REVIEW

File: samples/deploy/crd/v1/ApisixPluginConfig.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
     shortNames:
       - apc
   versions:
-    - name: v2beta2
+    - name: v2beta3
```
[the remark this answers is not in the document]
Comment (tao12345666333) [reply samples/deploy/crd/v1/ApisixPluginConfig.yaml:32]: Yes, the reason for the change here is #707 and https://github.com/apache/apisix-ingress-controller/pull/791#issuecomment-990706134

And this feature has not yet been provided to users (not really implemented). So the modification here is no problem.

**Comment by neverCase** [comment]
@tao12345666333 
Run e2e test cases failed again.
Does this problem cause by the ingress status bug which you mentioned before？

File: samples/deploy/crd/v1/ApisixPluginConfig.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tokers) [reply samples/deploy/crd/v1/ApisixPluginConfig.yaml:32]: OK.

**Review by tokers — APPROVED** [review]

**Comment by tao12345666333** [comment]
No. The ingress status bug has been fixed #788

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**